__tests__/transform/object.test.js

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+var transformObject = require('../../lib/transform/object');
+
+var options = {
+  transforms: [
+    {
+      type: 'attribute',
+      transformer: function() {
+        return {foo: 'bar'}
+      }
+    }, {
+      type: 'attribute',
+      transformer: function() {
+        return {bar: 'foo'}
+      }
+    }, {
+      type: 'name',
+      matcher: function(prop) {
+        return prop.attributes.foo === 'bar';
+      },
+      transformer: function() {
+        return "hello";
+      }
+    }
+  ]
+};
+
+describe('transform', () => {
+  describe('object', () => {
+    it('does not crash when called without parameters', () => {
+      expect(transformObject()).toEqual({});
+    })
+
+    it('returns expected object', () => {
+      const objectToTransform = {
+        "size": {
+          "font": {
+            "base": {
+              "value": "16",
+              "comment": "the base size of the font"
+            },
+            "large": {
+              "value": "20",
+              "comment": "the large size of the font"
+            }
+          }
+        },
+        "color": "#FFFF00"
```

Comment:
I think this should point to an object with a "value" key, like the other "tokens", or if you want to test that it outputs a property as is, maybe just use a "test" key with a "this is a test" string as value

Reply:
@didoo  is correct.